addons/interaction/functions/fnc_takeLauncher.sqf

```diff
@@ -0,0 +1,79 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Timi007. Cplhardcore
+ * Pass launcher to another unit.
+ *
+ * Arguments:
+ * 0: Unit that passes the launcher <OBJECT>
+ * 1: Unit to pass the launcher to <OBJECT>
+ * 2: Launcher classname <STRING>
+ *
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_player, _target, "launch_NLAW_F"] call ace_interaction_fnc_passLauncher
+ *
+ * Public: No
+ */
+params ["_player", "_target", "_launcher"];
+TRACE_3("Take launcher params",_player,_target,_launcher);
+
+private _playerName = [_player] call EFUNC(common,getName);
+private _cfgWeapons = configFile >> "CfgWeapons" >> _launcher;
+private _displayName = getText (_cfgWeapons >> "displayName");
+[QEGVAR(common,displayTextStructured), [[LSTRING(TakingLauncherHint), _playerName, _displayName], 1.5, _target], [_target]] call CBA_fnc_targetEvent;
+
+if (_launcher isEqualTo "") exitWith {ERROR("No launcher specified.")};
+if (secondaryWeapon _player != "") exitWith {ERROR("Cannot add launcher to target due to lack of inventory space.")};
```

Comment:
`isEqualTo` is already used right above for a similar purpose, so this one shouldn't be any different:
```suggestion
if (secondaryWeapon _player isNotEqualTo "") exitWith {ERROR("Cannot add launcher to target due to lack of inventory space.")};
```